cpp/include/cudf/strings/convert/convert_ipv4.hpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/strings/strings_column_view.hpp>
+#include <cudf/column/column.hpp>
+
+namespace cudf
+{
+namespace strings
+{
+
```

Comment:
need @throws docs for both of these.

Reply:
Only the `integers_to_ipv4` will throw due to `CUDF_EXPECTS` and it the doxygen already includes a `@throw` .
https://github.com/rapidsai/cudf/blob/3eb3ff829734b9165031efb51af0b36af173eccb/cpp/include/cudf/strings/convert/convert_ipv4.hpp#L66

Reply:
My mistake.